swap/Cargo.toml

```diff
@@ -48,10 +50,12 @@ structopt = "0.3"
 strum = { version = "0.20", features = ["derive"] }
 thiserror = "1"
 time = "0.2"
-tokio = { version = "1", features = ["rt-multi-thread", "time", "macros", "sync", "process", "fs"] }
+tokio = { version = "1", features = ["rt-multi-thread", "time", "macros", "sync", "process", "fs", "net"] }
+tokio-socks = "0.5"
 tokio-tungstenite = { version = "0.14", features = [ "rustls-tls" ] }
 tokio-util = { version = "0.6", features = ["io"] }
 toml = "0.5"
+torut = { version = "0.1", features = ["serialize", "v2", "v3", "control", "vendored_openssl"] }
```

Comment:
Is the traffic to the socks5 proxy encrypted or what do we need openssl for?

Onion v2 is EOL as far as I know, so if it is only for the RSA keys in Onion v2, then we should drop v2 support IMO. Not worth the added dependency baggage.

Reply:
Interesting, Torout seems to need openssl only for v2 keys. So dropping this should allow us to drop openssl :) Let's see what CI says.